test/e2e/network/ingressclass.go

```diff
@@ -96,6 +97,38 @@ var _ = common.SIGDescribe("IngressClass [Feature:Ingress]", func() {
 		}
 	})
 
+	ginkgo.It("should allow IngressClass to have Namespace-scoped parameters [Serial]", func() {
+		ingressClass := &networkingv1.IngressClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "ingressclass1",
+				Labels: map[string]string{
+					"ingressclass":  f.UniqueName,
+					"special-label": "generic",
+				},
+			},
+			Spec: networkingv1.IngressClassSpec{
+				Controller: "example.com/controller",
+				Parameters: &networkingv1.IngressClassParametersReference{
+					Scope:     utilpointer.StringPtr("Namespace"),
+					Namespace: utilpointer.StringPtr("foo-ns"),
+					Kind:      "fookind",
+					Name:      "fooname",
+					APIGroup:  utilpointer.StringPtr("example.com/v1"),
+				},
+			},
+		}
+		createdIngressClass, err := cs.NetworkingV1().IngressClasses().Create(context.TODO(), ingressClass, metav1.CreateOptions{})
+		if createdIngressClass.Spec.Parameters == nil {
+			framework.Failf("Expected IngressClass.spec.parameters to be set")
+		}
```

Comment:
the CI job fails here

> /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network/ingressclass.go:100
> Nov  9 01:49:18.288: Expected IngressClass.spec.parameters to be set
> /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:113

is the ingress version used in the CI supporting it? otherwise we have to modify the job in test-infra to skip this test

Reply:
> is the ingress version used in the CI supporting it?

I'm not sure if that even matters because the test creates an IngressClass resource (and not an Ingress resource).

I've fixed the test to include a more descriptive error message and fixed the ordering.
Another run _should_ point out the issues.